### DISCUSSION TITLE
Side Drawer with content sticky not working

### DISCUSSION THREAD

**Discussion by chadams** [saadeghi/daisyui#756] [Q&A] [answered]
I'm trying to Make a simple page of documentation. Sticky header, Footer, and Drawer on the left to hold the documentation outline, with main content on right. I can't seem to make the Drawer sticky, It scrolls with the right content. "sticky top-0" isn't working. I've copied the Drawer with mobile docking example from the site exactly.

**Comment by saadeghi** [comment, marked as answer]
Here's an example: https://play.tailwindcss.com/8mFBLFLF1j?size=1050x720  
Please provide a code example if you're having problem with your layout or if you want to make a different layout
